test/e2e/apps/framework.go

```diff
@@ -16,9 +16,51 @@ limitations under the License.
 
 package apps
 
-import "github.com/onsi/ginkgo"
+import (
+	"fmt"
+
+	"github.com/onsi/ginkgo"
+	appsv1 "k8s.io/api/apps/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/wait"
+	clientset "k8s.io/client-go/kubernetes"
+	appsclient "k8s.io/client-go/kubernetes/typed/apps/v1"
+	"k8s.io/kubernetes/test/e2e/framework"
+)
 
 // SIGDescribe annotates the test with the SIG label.
 func SIGDescribe(text string, body func()) bool {
 	return ginkgo.Describe("[sig-apps] "+text, body)
 }
+
+// WaitForReplicaSetDesiredReplicas waits until the replicaset has desired number of replicas.
+func WaitForReplicaSetDesiredReplicas(rsClient appsclient.ReplicaSetsGetter, replicaSet *appsv1.ReplicaSet) error {
```

Comment:
Thanks @tanjunchen 
Would you consider creating a new go file (such as `test/e2e/apps/wait.go`) to hold these code ?

Reply:
Sure. i will do it